ui/src/components/denali/InputDropdown.js

```diff
@@ -295,7 +295,7 @@ class InputDropdown extends React.Component {
                                     selected:
                                         options.selectedItem &&
                                         options.selectedItem.value ===
-                                        item.value,
+                                            item.value,
```

Comment:
Why do we change the indent here?

Reply:
the `npm run fix-lint` script is a bit funky and changes things back and forth